src/Raven.Client/Documents/Operations/Backups/GetPeriodicBackupStatusOperation.cs

```diff
@@ -52,5 +52,7 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
     public class GetPeriodicBackupStatusOperationResult
```

Comment:
```
AbstractGetPeriodicBackupStatusOperationResult
	bool IsSharded
	
GetPeriodicBackupStatusOperationResult : AbstractGetPeriodicBackupStatusOperationResult
    PeriodicBackupStatus Status;

GetShardedPeriodicBackupStatusOperationResult : AbstractGetPeriodicBackupStatusOperationResult
    PeriodicBackupStatus[] Statuses;
```

Let's introduce a separate operation for sharded one.

In regular you will check IsSharded and throw for sharded, in Sharded one you will throw for regular database

Reply:
done